.github/workflows/maven.yml

```diff
@@ -14,6 +14,7 @@ jobs:
       matrix:
         platform: [ ubuntu-latest ]
         java-version: [ 8 ]
+        profile: [ 1.10, 1.11, 1.12, 2.0, 2.1, 2.2, 2.3, 2.4 ]
 
```

Comment:
This is great.  Once this is committed I'd be in favor of removing all of these build stages from Bamboo since they will all be covered here, and it's a massive load (and delay) in Bamboo